mutox/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
```

Comment:
I would prefer adding the `mutox` folder within the `sonar/models` directory, next to other models: `blaser`, `sonar_text`, etc.